hr_holidays_compute_days/views/hr_holidays_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="edit_holiday_new" model="ir.ui.view">
+            <field name="name">Leave Request (hr_holidays_working_time)</field>
+            <field name="model">hr.holidays</field>
+            <field name="inherit_id" ref="hr_holidays.edit_holiday_new" />
+            <field name="arch" type="xml">
+                <field name="date_from" position="attributes">
+                    <attribute name="context">{'employee_id': employee_id, 'holiday_status_id': holiday_status_id}</attribute>
+                    <attribute name="on_change">onchange_date_from(date_to, date_from, context)</attribute>
+                </field>
+                <field name="date_to" position="attributes">
+                    <attribute name="context">{'employee_id': employee_id, 'holiday_status_id': holiday_status_id}</attribute>
+                    <attribute name="on_change">onchange_date_to(date_to, date_from, context)</attribute>
+                </field>
+                <field name="employee_id" position="attributes">
+                    <attribute name="context">{'date_from': date_from, 'date_to': date_to, 'holiday_status_id': holiday_status_id}</attribute>
+                    <attribute name="on_change">onchange_employee(employee_id, context)</attribute>
```

Comment:
New api don't needs `onchange` in view, just needs `api.depends` in method
Is this a special case where we need this onchange in view?